torch/fx/experimental/sym_node.py

```diff
@@ -630,6 +634,15 @@ def _sympy_abs(a):
     return sympy.Abs(a)
 
 
+def _sympy_round(number, ndigits=None):
+    from torch.utils._sympy.functions import Round
+
+    # FIXME: sympy doesn't allow passing None
+    args = [ndigits] if ndigits is not None else []
+
+    return Round(number, *args)
```

Comment:
The args are based on what gets passed to `eval`, and `None` is not something that SymPy has a symbolic version of. 

You could fix this by doing something like this

```py
_default = object()

class Round(Function):
    @classmethod
    def eval(cls, number, ndigits=_default):
        if ndigits is None:
            return Round(number)
```

This always normalizes ndigits=None to the one-element `args` variant. 

I think you could also fix it by overriding `__new__` and bypassing the logic for `None`, but I wouldn't necessarily recommend that as it would be a bit more complicated due to the way `Function` is implemented (see https://github.com/sympy/sympy/blob/master/sympy/core/function.py if you are interested).

Of course, just having two separate classes as you have here is also an option. It might make more sense to do this if you end up with a lot of logic that you'd like to share between the classes. But as long as that isn't the case, and you are OK with constructing the classes separately (or by using some sort of `round()` helper function), separate classes is fine.

Reply:
> or by using some sort of round() helper function

Of course, if you do this, you could also just handle the logic there, like

```py
def round(number, ndigits=None):
    if ndigits is None:
        return Round(number)
    return Round(number, ndigits)
```

The point is just that `None` is not a SymPy type, so you cannot pass it to a SymPy object.